arviz/plots/tsplot.py

```diff
@@ -283,8 +283,10 @@ def plot_ts(
     len_y = len(y_plotters)
     len_x = len(x_plotters)
     length_plotters = len_x * len_y
-    y_plotters = np.tile(y_plotters, (len_x, 1))
-    x_plotters = np.tile(x_plotters, (len_y, 1))
+    # TODO: Incompatible types in assignment (expression has type "ndarray[Any, dtype[Any]]",
+    # TODO: variable has type "List[Any]")  [assignment]
+    y_plotters = np.tile(y_plotters, (len_x, 1))  # type: ignore[assignment]
+    x_plotters = np.tile(x_plotters, (len_y, 1))  # type: ignore[assignment]
```

Comment:
this will lead to mypy errors. The error message has been shown, and likely this is due to newer numpy version, and similar typing issue has been seen in other dependencies. For now, can ignore the error to fix the CI.